pkg/github/github.go

```diff
@@ -34,14 +35,21 @@ type CreateIssueOptions struct {
 	Title          string   `json:"title,omitempty"`
 	UpdateExisting bool     `json:"updateExisting,omitempty"`
 	Token          string   `json:"token,omitempty"`
+	TrustedCerts   []string `json:"trustedCerts,omitempty"`
 }
 
 //NewClient creates a new GitHub client using an OAuth token for authentication
-func NewClient(token, apiURL, uploadURL string) (context.Context, *github.Client, error) {
-	ctx := context.Background()
-	ts := oauth2.StaticTokenSource(
-		&oauth2.Token{AccessToken: token},
-	)
+func NewClient(token, apiURL, uploadURL string, trustedCerts []string) (context.Context, *github.Client, error) {
+	httpClient := piperhttp.Client{}
+	httpClient.SetOptions(piperhttp.ClientOptions{
+		//TransportSkipVerification: true,
+		TrustedCerts: trustedCerts,
+		DoLogRequestBodyOnDebug: true,
```

Comment:
bind the setting to `verbose`?

Reply:
It is already bound to verbose flag since it outputs only once dbug log level is configured which is bound to verbose.